app/src/main/res/values/component_colors.xml

```diff
@@ -289,6 +290,8 @@
   <color name="component_color_audio_fragment_seekbar_color">@color/color_palette_seekbar_progress_background_color</color>
   <!-- Topic Revision Fragment -->
   <color name="component_color_topic_revision_fragment_background_color">@color/color_palette_topic_revision_background_color</color>
+   <!-- Topic Lesson Fragment -->
```

Comment:
@iampradeep-hr if this is not part of your issue, so remove this color also.

Reply:
removed it